ee-feature-pack/galleon-shared/src/main/resources/modules/system/layers/base/org/wildfly/transaction/client/main/module.xml

```diff
@@ -7,6 +7,10 @@
 
 <module xmlns="urn:jboss:module:1.9" name="org.wildfly.transaction.client">
 
+    <properties>
+        <property name="jboss.api" value="unsupported"/>
```

Comment:
Unsupported is a downstream-only setting. It means "'public' upstream but 'private' downstream" so it should never be used upstream.